assets/js/search.js

```diff
@@ -28,7 +28,7 @@
   /**
    * Function displays the total items displayed on the page
    */
-  function showItemsCount(totalCount, numSubmodules) {
+  function showItemsCount(searchEntry, totalCount, numSubmodules) {
```

Comment:
`searchEntry` isn't used here?

Reply:
What are you talking about? It's in the line just below this one

Reply:
Oh... So how did this work at all before? I saw this variable added and figured it was new and was expecting usage of it to show up in the diff too... Was the code just throwing errors all the time instead?

Reply:
It was working because `searchEntry` was a global var set at the inital page load. I removed the global var and placed it under beter scoping.

Reply:
Ohhh, understood, thx